sfm/ui/templates/ui/collection_detail.html

```diff
@@ -120,10 +121,11 @@ <h1 class="panel-title">{{ collection.name }}</h1>
         {% if collection.stats %}
             <p><strong>Stats:</strong><ul>
                 {% for item, count in collection.stats.items %}
-                    <li>{{item}}: {{count}}</li>
+                    <li>{{ item }}: {{ count|intcomma }}</li>
                 {% endfor %}
             </ul></p>
         {% endif %}
+        <p><strong>WARCs:</strong> {{ collection.warcs_count }} file{{ collection.warcs_count|pluralize }} ({{ collection.warcs_bytes|filesizeformat }})</p>
```

Comment:
If there are zero files because the first harvest hasn't completed yet, the UI shows: "WARCs: None file (0 bytes)". Can this be "WARCs: 0 files (0 bytes)" instead?